_posts/2021-03-25-Creating-Memory-Game.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Creating a Memory Game 
+description: 
+summary: 
+tags:
+---
+Today, we completed an hour on typing club and as usual we sent an image of our progress. Then, Kajari sent us our task for the day. We had to create a memory game, which was the first out of 7 games on a youtube tutorial. To get started we created a folder called 'Memory-game', then we downloaded the given images and put them in a folder called 'images'. I then put the images folder inside of the Memory game folder, which allowed me to open all the files on Sublime Text. Then, I followed the youtube tutorial and wrote down the code for each of the folders. In the youtube tutorial, she explains step by step how to complete the game. From the youtube tutorial I learned that on the index.html file our game grid was created, and on the same file we have to link our css file. Also, when we started on  the app.js file, the first thing we have to do was to name the cards, which was the name we were going to use throughout the file. I also learned that in the app.js file we had to link many things so, that the computer was could see that we were referencing the images. Lastly, I saved the code and went on Chrome to check if the game worked. [This is the link we used](https://www.youtube.com/watch?v=lhNdUVh3qCc) When I was done with the game I had to go to Github and make a new repository. [This is the final product](https://celestem406.github.io/Memory_Game/)
+The functions that we learned this week were:
+* push() - allowsto add new items at the end of the array, while also changing the length of the array.
+* queryselector() - returns the first element from the document that match a group of selectors, or the specific selector.
```

Comment:
Add that the document is the DOM which stands for "document object model" at the end of this line.